### ISSUE TITLE
Resize widget when uploader dialog is closed or canceled.

### ISSUE THREAD

**Issue by kennethpdev** [vinodc/jotform-cloud-storage#2]
Hi, I'm a developer at JotForm, this is regarding to the Kloudless File Uploader widget.

Is there a way to listen to the "cancel" and "close" event of the uploader? What I mean is that when you close the dialog it should have resize the widget to its original size.

---
Also  I made a PR #1 to address the scrollbar issue on card forms. In case you are in MIA state we will copy the content of this repo and serve it under our platform to make some customizations.

Regards,

**Comment by vinodc** [comment]
Hi. Yes there are events that can be listened to as documented here: https://github.com/kloudless/file-explorer#events

Thanks for the PR.

**Comment by kennethpdev** [comment]
Yes please if you can, update the widget to resize the widget frame whenever a user close or cancel the uploader dialog.

**Comment by vinodc** [comment]
@kenma9123 I'm not entirely sure of what you mean, but i will gladly accept any PRs regarding that if you believe it is the best user experience.
